main.nf

```diff
@@ -260,11 +185,15 @@ if (params.input_paths) {
         .into { ch_read_files_fastqc; ch_read_files_trimming }
 }
 
+////////////////////////////////////////////////////
+/* --         PRINT PARAMETER SUMMARY          -- */
+////////////////////////////////////////////////////
+log.info NfcoreSchema.params_summary_log(workflow, params, json_schema)
+
 // Header log info
-log.info nfcoreHeader()
 def summary = [:]
 if (workflow.revision) summary['Pipeline Release'] = workflow.revision
-summary['Run Name']  = custom_runName ?: workflow.runName
+summary['Run Name']  = workflow.runName
```

Comment:
Do we still need to build this summary object if we have the function to print the log at the start of the run? I guess it's still used in the email for now..? 🤔

Reply:
Yup, it's part of the email

Reply:
Ok, should strip that in the next tools release then I guess :)